ISSUE_TEMPLATE.md

```diff
@@ -1,22 +0,0 @@
-<!--
```

Comment:
while you're at it you could move the CODEOWNERS to the .github folder as well. And maybe CONTRIBUTING.md too?

Reply:
I didn't want to move that one as I don't know if I need to change the paths and secondly, CODEOWNERS seems to be useful on its own